### DISCUSSION TITLE
AOE spell for Rogue/Feral Druid

### DISCUSSION THREAD

**Discussion by K1PCKhor** [chromiecraft/chromiecraft#4055] [Suggestions]
Greetings;
I was thinking maybe adding Fan of Knives and Swipe(feral) maybe a good thing. These 2 classes have no AOE, and they actually are percentage based spells (no flat damage, so they become broken)!

**Comment by BeefSteaker** [comment]
I think it's unlikely this would be Implemented because it's not blizzlike, and would require custom changes

**Comment by K1PCKhor** [comment]
A lot of things aren't Blizzlike, It's a change that makes these classes a bit more enjoyable to play (and more viable) and it won't affect other game experiences!

**Comment by Never6** [comment]
As a feral main if they were gonna go the route of giving us spells before we are supposed too I would much rather have savage  roar before swipe.
